src/main/java/MorseCode.java

```diff
@@ -0,0 +1,121 @@
+        morseCharacters.put('-', "-....-");
+        morseCharacters.put('/', "-..-.");
+        morseCharacters.put('"', ".-..-.");
+    }
+
+    public void addOtherDirection()
+    {
+        morseCharactersOtherDirection.put(".-", 'A');
+        morseCharactersOtherDirection.put("-...", 'B');
+        morseCharactersOtherDirection.put("-.-.", 'C');
+        morseCharactersOtherDirection.put("-..", 'D');
+        morseCharactersOtherDirection.put(".", 'E');
+        morseCharactersOtherDirection.put("..-.", 'F');
+        morseCharactersOtherDirection.put("--.", 'G');
+        morseCharactersOtherDirection.put("....", 'H');
+        morseCharactersOtherDirection.put("..", 'I');
+        morseCharactersOtherDirection.put(".---", 'J');
+        morseCharactersOtherDirection.put("-.-", 'K');
+        morseCharactersOtherDirection.put(".-..", 'L');
+        morseCharactersOtherDirection.put("--", 'M');
+        morseCharactersOtherDirection.put("-.", 'N');
+        morseCharactersOtherDirection.put("---", 'O');
+        morseCharactersOtherDirection.put(".--.", 'P');
+        morseCharactersOtherDirection.put("--.-", 'Q');
+        morseCharactersOtherDirection.put(".-.", 'R');
+        morseCharactersOtherDirection.put("...", 'S');
+        morseCharactersOtherDirection.put("-", 'T');
+        morseCharactersOtherDirection.put("..-", 'U');
+        morseCharactersOtherDirection.put("...-", 'V');
+        morseCharactersOtherDirection.put(".--", 'W');
+        morseCharactersOtherDirection.put("-..-", 'X');
+        morseCharactersOtherDirection.put("-.--", 'Y');
+        morseCharactersOtherDirection.put("--..", 'Z');
+        morseCharactersOtherDirection.put("-----", '0');
+        morseCharactersOtherDirection.put(".----", '1');
+        morseCharactersOtherDirection.put("..---", '2');
+        morseCharactersOtherDirection.put("...--", '3');
+        morseCharactersOtherDirection.put("....-", '4');
+        morseCharactersOtherDirection.put(".....", '5');
+        morseCharactersOtherDirection.put("-....", '6');
+        morseCharactersOtherDirection.put("--...", '7');
+        morseCharactersOtherDirection.put("---..", '8');
+        morseCharactersOtherDirection.put("----.", '9');
+        morseCharactersOtherDirection.put(".-.-.-", '.');
+        morseCharactersOtherDirection.put("--..--", ',');
+        morseCharactersOtherDirection.put("---...", ':');
+        morseCharactersOtherDirection.put("..--..", '?');
+        morseCharactersOtherDirection.put(".----.", '\'');
+        morseCharactersOtherDirection.put("-....-", '-');
+        morseCharactersOtherDirection.put("-..-.", '/');
+        morseCharactersOtherDirection.put(".-..-.", '"');
+    }
+
+    public HashMap<Character, String> getMorseCharacters()
+    {
+        return this.morseCharacters;
+    }
+
+    public HashMap<String, Character> getMorseCharactersOtherDirection()
+    {
+        return this.morseCharactersOtherDirection;
+    }
```

Comment:
Instead of providing getters for the hashmaps, provide two methods. One to translate to Morse, the other to translate to English. This is better because if you decide to change how the data is stored, you don't need to change how this class is used.